src/pudl/metadata/codes.py

```diff
@@ -1564,6 +1564,8 @@
             "WT": "WND",
             "H2": "OG",
             "OOG": "OG",
+            "Sun": "SUN",
+            "sub": "SUB",
```

Comment:
We could probably write a function that would just turn this column uppercase and avoid this problem in the future. I don't think we have a helper function that does that right now. Maybe it's too simple for a helper function? Thoughts on where this type of generalized solution to this spot fix should go? @zaneselvans

Reply:
Encoder

Reply:
Is there an instance when we wouldn't want to apply it that you can think of?

Reply:
I think I'm gonna make another PR for this